### ISSUE TITLE
Creative Fluid Tank craft isnt working

### ISSUE THREAD

**Issue by DrDeucecom** [Cazadorsniper/PO3#1740]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:** 
**Pack Mode (Normal, Titan, Kappa, Mythic):** Normal Mode
**Singleplayer, Multiplayer, or LAN?:** 
3.4.7
Normal Mode
Single
**Description:**
With the correct recipe in the ultimate crafting table the creative fluid tank is unable to be crafted and it is not showing up where the item should be.
**Steps to reproduce:**

**Crash log (use pastebin or issue will be closed):**
![Creative tank recipe](https://user-images.githubusercontent.com/128867682/227663813-03a3f0bf-add0-4c89-8a53-93af8632c636.png)

**Comment by Cazadorsniper** [comment]
Are you sure you have the correct recipe? All neutral steels look exactly the same.

**Comment by DrDeucecom** [comment]
Yes, it is the correct recipe with the 11th neutral steel. I've tried with the 10th but still no change.
